Remarkable-Donkeys/bangazonCLI
# Kn customers

# Steps To Test
1. `git fetch --all`
2. `git checkout kn-customers`
3. `cd test`
4. `dotnet test`
5. make sure that all the tests pass: AddNewCustomer, ActiveCustomer

[New Customer](https://github.com/Remarkable-Donkeys/bangazonCLI/issues/1)
[Active Customer](https://github.com/Remarkable-Donkeys/bangazonCLI/issues/2)

### Description of what should happen when steps are followed
Both tests should pass and a new customer should be added to your database. 

Also, on CustomerManager, ActiveCustomerId has been changed to a public static int so all you need to do to access the active customer id is invoke CustomerManager.ActiveCustomerId. Please change any tests that are accessing the GetActive() method to CustomerManager.ActiveCustomerId so that we can delete the GetActive() method since it is unnecessary now.

**At least one person needs to approve pull request before merged to master**
